rclpy/test/test_executor.py

```diff
@@ -136,6 +136,23 @@ def test_multi_threaded_executor_executes(self):
         finally:
             executor.shutdown()
 
+    def test_multi_threaded_executor_closes_threads(self):
+        self.assertIsNotNone(self.node.handle)
+
+        def get_threads():
+            return set(t.name for t in threading.enumerate())
```

Comment:
can you fix https://ci.ros2.org/job/ci_linux/16308/testReport/junit/(root)/projectroot/flake8/ error?

```
-- run_test.py: invoking following command in '/home/jenkins-agent/workspace/ci_linux/ws/src/ros2/rclpy/rclpy':
 - /home/jenkins-agent/workspace/ci_linux/ws/install/ament_flake8/bin/ament_flake8 --xunit-file /home/jenkins-agent/workspace/ci_linux/ws/build/rclpy/test_results/rclpy/flake8.xunit.xml
            return set(t.name for t in threading.enumerate())
                   ^
1     C401 Unnecessary generator - rewrite as a set comprehension.

./test/test_executor.py:143:20: C401 Unnecessary generator - rewrite as a set comprehension.
```

Reply:
This should be passing now